README.md

```diff
@@ -23,10 +23,44 @@
 
 <p><img src="schedule.png" alt="LTS Schedule"/></p>
 
-The LTS Schedule is available also as a [JSON][] file or [ICal][]. There is
+The Release schedule is available also as a [JSON][] file or [ICal][]. There is
 also a live [Google Calendar][] that may be subscribed to.
 
-# LTS Plan
+## Mandate
+
+The Release working group's purpose is:
+
+* Management/execution of the release process for all releases.
```

Comment:
`Management/execution of the release and support process for all Node.js releases.`